README.md

```diff
@@ -30,5 +30,5 @@ See [User Guide](http://docs.javaslang.io)
 Currently only basic microbenchmarks are available. To run
 
 ```
-mvn clean test -Pbenchmark
+mvn clean test -pl javaslang-benchmark
```

Comment:
The profile doesn't exist anymore, so I update the docs to run the benchmarks only.
`pl` stands for project list I guess.